### PR TITLE
Measurements must use token validation

### PR DESCRIPTION
**Pull request by sahma19**

### CODE REVIEW

File: domains/measurements/Measurements.API/API/Controllers/MeasurementsController.cs [hunk 1]
```diff
@@ -1,54 +1,60 @@
...
 [ApiController]
 [Authorize]
-public class MeasurementsController : AuthorizationController
+public class MeasurementsController : ControllerBase
 {
+
     [HttpGet]
     [Route("api/measurements/consumption")]
-    public async Task<ActionResult<MeasurementResponse>> GetConsumptionMeasurements([FromQuery] MeasurementsRequest request, IMeasurementsService measurementsService, IValidator<MeasurementsRequest> validator)
+    public async Task<ActionResult<MeasurementResponse>> GetConsumptionMeasurements([FromQuery] MeasurementsRequest request, IMeasurementsService measurementsService, IValidator<MeasurementsRequest> validator, [FromServices] IHttpContextAccessor httpContextAccessor)
```
Comment (tnickelsen) [inline domains/measurements/Measurements.API/API/Controllers/MeasurementsController.cs:19]: Did you test that it works?

File: domains/measurements/Measurements.API/API/Models/Measurement.cs [hunk 2]
```diff
@@ -1,6 +1,5 @@
 namespace API.Models;
 
-//NOTE: This DTO is a reflection of the output from the DataSync API.
```
Comment (tnickelsen) [inline domains/measurements/Measurements.API/API/Models/Measurement.cs:3]: Why delete this?

File: domains/measurements/Measurements.API/Tests/AggregationTests.cs [hunk 3]
```diff
@@ -20,16 +20,11 @@ public sealed class AggregationTests
     [InlineData(Aggregation.Year, new long[] { 1609538400 }, new long[] { 1609552799 }, new long[] { 3930 })]
     public void Measurements_CalculateAggregation_ForConsumption(Aggregation aggregation, long[] expectedDateFrom, long[] expectedDateTo, long[] expectedValues)
     {
-        // Arrange
-        var dateFrom = new DateTimeOffset(2021, 1, 1, 22, 0, 0, TimeSpan.Zero);
-        var dateTo = new DateTimeOffset(2021, 1, 2, 1, 59, 59, TimeSpan.Zero);
```
Comment (tnickelsen) [inline domains/measurements/Measurements.API/Tests/AggregationTests.cs:25]: Why delete these?

File: domains/measurements/Measurements.API/API/Controllers/MeasurementsController.cs (hunk 1 above)
Comment (sahma19) [reply domains/measurements/Measurements.API/API/Controllers/MeasurementsController.cs:19]: Yes, I have spun up a preview environment, and it works.

File: domains/measurements/Measurements.API/Tests/AggregationTests.cs (hunk 3 above)
Comment (sahma19) [reply domains/measurements/Measurements.API/Tests/AggregationTests.cs:25]: The tests don't even use the variables. They declare them, but aren't being used whatsoever.

File: domains/measurements/Measurements.API/API/Models/Measurement.cs (hunk 2 above)
Comment (sahma19) [reply domains/measurements/Measurements.API/API/Models/Measurement.cs]: I added it back: https://github.com/Energinet-DataHub/energy-origin/pull/539/commits/a4abb707214598c4c2adb390115852b1fd970f81

**Pull request closed without merging**